boggle.js

```diff
@@ -11,21 +11,134 @@ function newBoggleBoard() {
  * Prints out a boggle board.
  */
 function printBoggleBoard(boggleBoard) {
+  let includesQu = false;
+  for (let i = 0; i < boggleBoard.length; i++) {
+    for (let j = 0; j < boggleBoard[i].length; j++) {
+      if (boggleBoard[i][j] === 'Q') {
+        boggleBoard[i][j] = 'Qu';
+        includesQu = true;
+      }
+    }
+  }
+
+  if (includesQu) {
+    for (let i = 0; i < boggleBoard.length; i++) {
+      for (let j = 0; j < boggleBoard[i].length; j++) {
+        boggleBoard[i][j] = boggleBoard[i][j].padEnd(2);
```

Comment:
Don't modify the board itself!

This logic only happens at the display level. Write a function that takes a symbol and input (representing board state) and returns the appropriate display string (padded letter or Qu).

Use it when printing.